src/main/java/fi/hsl/transitdata/tripupdate/gtfsrt/GtfsRtFactory.java

```diff
@@ -76,6 +76,7 @@ public static long lastModified(InternalMessages.StopEstimate estimate) {
                 .setDirectionId(direction)
                 .setStartDate(estimate.getTripInfo().getOperatingDay()) // Local date as String
                 .setStartTime(estimate.getTripInfo().getStartTime()) // Local time as String
+                .setScheduleRelationship(GtfsRealtime.TripDescriptor.ScheduleRelationship.SCHEDULED)
```

Comment:
Was this just forgotten or why is the `ScheduledRelationship` added?

Reply:
it seems we've never had it so probably forgotten..? the field is optional so it wasn't a bug but somehow it feels appropriate to add it..?